cmd/zfs/zfs_main.c

```diff
@@ -4943,6 +4943,7 @@ zfs_do_receive(int argc, char **argv)
 #define	ZFS_DELEG_PERM_MOUNT		"mount"
 #define	ZFS_DELEG_PERM_SHARE		"share"
 #define	ZFS_DELEG_PERM_SEND		"send"
+#define	ZFS_DELEG_PERM_SEND_RAW		"send-raw"
```

Comment:
To separate this from other subcommands, do we maybe want to call it `send:raw` or something? Creating a namespace under the subcommand for these parameter based restrictions?